docs/UserGuide.md

```diff
@@ -781,4 +794,75 @@ Something goes here...
 
 ## Glossary
```

Comment:
Perhaps, including some of the technical terms present will be useful:

- Command line
- syntax
- parameters/arguments 

These are non-exhaustive. Add more if you think I have missed out any. If you think they have been specified clearly enough in the UG, it is fine to leave them out.